gtk4/src/lib.rs

```diff
@@ -14,6 +14,16 @@
 #![allow(deprecated)]
 
 pub use ffi;
+// Re-export gtk dependencies
```

Comment:
Only crates that defines types that are used in the public API IMHO, not all dependencies. But I think that's the case here?

Reply:
Well, yes, those are the gtk dependencies that are part of the public API.

Reply:
graphene/cairo are used for snapshot stuff along with gsk. gdk_pixbuf is still used on cell renders with a promise to be gone in gtk5 :)

Reply:
Ok :)